src/main/java/dream/fcard/gui/controllers/windows/MainWindow.java

```diff
@@ -204,4 +209,18 @@ public void openEditor(boolean isJava) {
 
     }
 
+    /**
+     * Opens a new window to show the user's statistics.
+     */
+    @FXML
+    public void openStatistics() {
+        System.out.println("Open statistics window");
+
+        Stage stage = new Stage();
+        stage.initModality(Modality.APPLICATION_MODAL);
+        Scene scene = new Scene(new StatisticsWindow());
+        stage.setScene(scene);
+        stage.setTitle("Your Statistics");
```

Comment:
Non-technical suggestion, maybe "My Statistics" would put the user in the center of the UI experience more than "Your Statistics"